english/sleep/more-sleep/smore-eating/delicious.md

```diff
@@ -8,9 +8,10 @@ When it catches fire, you blow it out.
 The result looks a little strange, but it is just as delicious as you could have imagined. Afterward, all you can
 think about is taking a big nap.
 
+However, then you think is napping such a good idea?  Isn't napping the thing that started this whole bizzare adventure?
+
 You decide to:
 
 [Take that nap](../../marshmallow.md)
 
 [Stay awake and go explore outside](../../../explore-outside/explore-outside.md)
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.